superbuild/cmake/FindImgui.cmake

```diff
@@ -0,0 +1,20 @@
+message("FindImgui CMAKE")
+message("CMAKE_INSTALL_PREFIX ${CMAKE_INSTALL_PREFIX}")
+
+unset(${Imgui_INCLUDE_DIR} CACHE )
+
+find_path(Imgui_INCLUDE_DIR
+        NAMES
+		imgui.h
+		HINTS
+        ${CMAKE_INSTALL_PREFIX}/include/imgui/
+)
+
+message(STATUS "Imgui_INCLUDE_DIR  ${Imgui_INCLUDE_DIR}")
+
```

Comment:
all of these superbuild/cmake/..cmake files are pretty repetetive - are they autogenerated? manually maintained?

Reply:
Manually maintained. Those files arent suppose to change anytime soon.